conda.recipe/meta.yaml

```diff
@@ -54,15 +58,15 @@ test:
     - mock
     - pandas
     - pytest >=2.9.0
-    - pytest-cov ==1.8.1
-    - pytest-selenium >=1.2.1
+    - pytest-cov ==1.8.1       [unix]
```

Comment:
Are we using `unix` here because win packages are not available in our Bokeh space? If that's the case, we should made and upload the packages...